iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/ShuntCompensatorImpl.java

```diff
@@ -102,7 +102,10 @@ public double getCurrentB() {
         return bPerSection * getCurrentSectionCount();
     }
 
-    @Override
+    /**
+     * @deprecated Use {@link #getbPerSection()} * {@link #getMaximumSectionCount()} instead.
+     */
+    @Deprecated
     public double getMaximumB() {
         return bPerSection * maximumSectionCount;
     }
```

Comment:
As there is a default implementation that is totally equivalent, you can simply remove this method to prepare the removal of the getMaximumB method.